scripts/gufi_stats

```diff
@@ -1088,8 +1103,12 @@ CUMULATIVE = [
 ]
 
 BOTH = [
+    ['dirsize-log2-bins',     dirsize_log2_bins],
+    ['dirsize-log1024-bins',  dirsize_log1024_bins],
     ['filesize-log2-bins',    filesize_log2_bins],
     ['filesize-log1024-bins', filesize_log1024_bins],
+    ['direntries-log2-bins',  direntries_log2_bins],
+    ['direntries-log1024-bins', direntries_log1024_bins],
```

Comment:
group the `dir*` stuff together

Reply:
As discussed, I'll just get rid of dirsize.